### PR TITLE
Set up associations

### PR DESCRIPTION
**Pull request by kwambiee**
Create Book, Classroom and rental class.
Used association to display relationships between the classes.

### CODE REVIEW

File: book.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```
Comment (uwadonat) [inline book.rb:9]: - [ ] Let's add a method for adding rentals. the book itself should be sent as a parameter to create the new rental achieving the `has-many` association.

File: person.rb [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+  attr_reader :id, :rentals
   attr_accessor :name, :age, :parent_permission
 
   def initialize(*args)
     @id = Random.rand(1..1000)
     @age = args[0]
     @name = args[1] || 'unknown'
     @parent_permission = args[2].nil? ? true : args[2]
+    @rentals = []
     super()
```
Comment (uwadonat) [inline person.rb]: - [ ] You should add a method for adding rentals. Here the person itself should be sent as a parameter to create the new rental achieving the `has-many` association.

**Review by uwadonat — CHANGES_REQUESTED** [review]
## Hi @kwambiee,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
- Your commit messages are very descriptive ✔️
- Your project is well organized and clean ✔️
- You wrote a professional readme file ✔️
- No linter errors ✔️
almost

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @kwambiee, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**